### PR TITLE
Next 12 Upgrade

### PR DESCRIPTION
**Pull request by noisekit**
- [x] Upgraded next-related deps
- [x] Cleaned up all the unnecessary dependencies related to babel (replaced with SWC)
- [x] Updated Jest configs (note the svg mocking - not amazing, but quick solution for now)

Amazing compilation speed with SWC

image

Jest is sped up as well!

Only some minor changes were necessary for configs and image imports, but rest is all good

Overall bundle size decreased significantly. Looks like the cumulative effect of webpack 5 + SWC eliminated a lot of duplicate code

## PREVIEW

https://staking-git-noisekit-synthetixio.vercel.app/

image

### CODE REVIEW

**Review by jmzwar — APPROVED** [review]
This looks straightforward, not too many configurations needed to change. I'd wait for @bachstatter to have a look before merging.

Excited for the performance improvement.

File: next.config.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ function optimiseContracts(config, { webpack }) {
 }
 
 module.exports = withPlugins([withBundleAnalyzer], {
+	swcMinify: true,
```
Comment (bachstatter) [inline next.config.js:34]: Do we want to minify in dev too?

**Review by bachstatter — APPROVED** [review]
Nice!!

**Comment by cavalier-eth** [comment]
@noisekit be nice to know the before and after speed or load times

**Pull request merged**